app/libs/android-auth-master/.editorconfig

```diff
@@ -0,0 +1,25 @@
+# EditorConfig helps developers define and maintain consistent
```

Comment:
The commit name for this patch doesn't seem to match. This change looks to be adding android auth master, rather than "created User class, UserService to get user data from API and put that into a User object"